pkgs/by-name/ho/holo-cli/package.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  protobuf,
+  pcre2,
+  unstableGitUpdater,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "holo-cli";
+  version = "0.4.0-unstable-2025-04-01";
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+  };
```

Comment:
Upstream now includes a lock file in version `0.5.0`, so we can update this and remove the lockfile.